<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <template class="GisConfirmPage" parent="GisPage">
    <child>
      <object class="GtkBox" id="page">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="valign">center</property>
        <property name="margin_left">64</property>
        <property name="margin_right">64</property>
        <property name="orientation">vertical</property>
        <property name="spacing">16</property>
        <child>
          <object class="GtkGrid" id="computer_grid">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="halign">center</property>
            <property name="hexpand">True</property>
            <property name="row_spacing">5</property>
            <property name="column_spacing">12</property>
            <child>
              <object class="GtkLabel" id="vendor_caption">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">end</property>
                <property name="hexpand">False</property>
                <property name="label" translatable="yes">Computer Vendor</property>
                <property name="xalign">1</property>
                <accessibility>
                  <relation type="label-for" target="vendor_label"/>
                </accessibility>
                <style>
                  <class name="dim-label"/>
                </style>
              </object>
              <packing>
                <property name="left_attach">0</property>
                <property name="top_attach">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="vendor_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">start</property>
                <property name="hexpand">True</property>
                <property name="label">To Be Filled By OEM</property>
                <property name="xalign">0</property>
              </object>
              <packing>
                <property name="left_attach">1</property>
                <property name="top_attach">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="product_caption">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">end</property>
                <property name="hexpand">False</property>
                <property name="label" translatable="yes">Product</property>
                <property name="xalign">1</property>
                <accessibility>
                  <relation type="label-for" target="product_label"/>
                </accessibility>
                <style>
                  <class name="dim-label"/>
                </style>
              </object>
              <packing>
                <property name="left_attach">0</property>
                <property name="top_attach">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="product_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">start</property>
                <property name="hexpand">True</property>
                <property name="label">To Be Filled By OEM</property>
                <property name="xalign">0</property>
              </object>
              <packing>
                <property name="left_attach">1</property>
                <property name="top_attach">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkGrid" id="image1_grid">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="halign">center</property>
            <property name="hexpand">True</property>
            <property name="row_spacing">5</property>
            <property name="column_spacing">12</property>
            <child>
              <object class="GtkLabel" id="image1_caption">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">end</property>
                <property name="hexpand">False</property>
                <property name="label" translatable="yes">Image</property>
                <property name="xalign">1</property>
                <style>
                  <class name="dim-label"/>
                </style>
              </object>
              <packing>
                <property name="left_attach">0</property>
                <property name="top_attach">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="image1_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">start</property>
                <property name="hexpand">True</property>
                <property name="label">eos-eos3.3-amd64-amd64.180115-104625.en.img.gz</property>
                <property name="xalign">0</property>
              </object>
              <packing>
                <property name="left_attach">1</property>
                <property name="top_attach">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="model1_caption">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">end</property>
                <property name="hexpand">False</property>
                <property name="label" translatable="yes">Target</property>
                <property name="xalign">1</property>
                <style>
                  <class name="dim-label"/>
                </style>
              </object>
              <packing>
                <property name="left_attach">0</property>
                <property name="top_attach">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="model1_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">start</property>
                <property name="hexpand">True</property>
                <property name="label">LITEON LGH-512V2G-11 M.2 2280 512GB</property>
                <property name="xalign">0</property>
              </object>
              <packing>
                <property name="left_attach">1</property>
                <property name="top_attach">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="device1_caption">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">end</property>
                <property name="hexpand">False</property>
                <property name="label" translatable="yes">Device</property>
                <property name="xalign">1</property>
                <style>
                  <class name="dim-label"/>
                </style>
              </object>
              <packing>
                <property name="left_attach">0</property>
                <property name="top_attach">2</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="device1_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">start</property>
                <property name="hexpand">True</property>
                <property name="label">/dev/sda</property>
                <property name="xalign">0</property>
              </object>
              <packing>
                <property name="left_attach">1</property>
                <property name="top_attach">2</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="size1_caption">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">end</property>
                <property name="hexpand">False</property>
                <property name="label" translatable="yes">Size</property>
                <property name="xalign">1</property>
                <style>
                  <class name="dim-label"/>
                </style>
              </object>
              <packing>
                <property name="left_attach">0</property>
                <property name="top_attach">3</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="size1_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="halign">start</property>
                <property name="hexpand">True</property>
                <property name="label">512 GB</property>
                <property name="xalign">0</property>
              </object>
              <packing>
                <property name="left_attach">1</property>
                <property name="top_attach">3</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkBox" id="warning_box">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <child>
              <object class="GtkImage" id="warning_icon">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="stock">gtk-dialog-warning</property>
                <property name="margin">16</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="warning_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="label" translatable="yes">All data on this target disk will be destroyed. You will lose &lt;b&gt;all of your apps, files and documents&lt;/b&gt;.</property>
                <property name="use_markup">True</property>
                <property name="wrap">True</property>
                <property name="wrap_mode">word-char</property>
                <property name="max_width_chars">50</property>
                <property name="xalign">0</property>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">2</property>
          </packing>
        </child>
        <child>
          <object class="GtkBox" id="wrong_computer_box">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <child>
              <object class="GtkImage" id="wrong_computer_icon">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="stock">gtk-dialog-warning</property>
                <property name="margin">16</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="wrong_computer_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="label" translatable="yes">Configured computer vendor and product information does not match this computer.</property>
                <property name="wrap">True</property>
                <property name="wrap_mode">word-char</property>
                <property name="max_width_chars">50</property>
                <property name="xalign">0</property>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">3</property>
          </packing>
        </child>
        <child>
          <object class="GtkLabel" id="countdown_label">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="label">Installation will commence automatically in 30 seconds.</property>
            <attributes>
              <attribute name="weight" value="bold"/>
              <attribute name="foreground" value="#efef29292929"/>
            </attributes>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">4</property>
          </packing>
        </child>
        <child>
          <object class="GtkButtonBox" id="cancel_continue_box">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="layout_style">spread</property>
            <child>
              <object class="GtkButton" id="cancel_button">
                <property name="label" translatable="yes">_Cancel</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="can_default">True</property>
                <property name="receives_default">True</property>
                <property name="use_underline">True</property>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="continue_button">
                <property name="label" translatable="yes">C_ontinue and Destroy Data</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="use_underline">True</property>
                <style>
                  <class name="destructive-action"/>
                </style>
              </object>
              <packing>
                <property name="expand">True</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">5</property>
          </packing>
        </child>
      </object>
    </child>
  </template>
  <object class="GtkSizeGroup" id="caption_size_group">
    <widgets>
      <widget name="vendor_caption"/>
      <widget name="product_caption"/>
      <widget name="image1_caption"/>
      <widget name="model1_caption"/>
      <widget name="device1_caption"/>
      <widget name="size1_caption"/>
    </widgets>
  </object>
  <object class="GtkSizeGroup" id="value_size_group">
    <widgets>
      <widget name="vendor_label"/>
      <widget name="product_label"/>
      <widget name="image1_label"/>
      <widget name="model1_label"/>
      <widget name="device1_label"/>
      <widget name="size1_label"/>
    </widgets>
  </object>
</interface>
